google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/ConnectionOptionsTest.java

```diff
@@ -151,21 +154,19 @@ public void testBuildWithAutoConfigEmulator() {
 
   @Test
   public void testBuildWithRouteToLeader() {
+    final String BASE_URI =
+        "cloudspanner:/projects/test-project-123/instances/test-instance-123/databases/test-database-123";
     ConnectionOptions.Builder builder = ConnectionOptions.newBuilder();
-    builder.setUri(
-        "cloudspanner:/projects/test-project-123/instances/test-instance-123/databases/test-database-123?routeToLeader=false");
+    builder.setUri(BASE_URI + "?routeToLeader=false");
```

Comment:
Nit: Prefer StringBuilder() . `+` is a more expensive way of concatenating strings.

Reply:
For single statement, the compiler would use `StringBuilder` automatically: https://stackoverflow.com/a/4645155.